sdk/analysisservices/Azure.ResourceManager.AnalysisServices/CHANGELOG.md

```diff
@@ -2,9 +2,17 @@
 
 ## 1.0.0-beta.1 (Unreleased)
```

Comment:
Here we have three versions including `1.0.0-beta.1`, `1.0.0` and `1.1.0-beta.1`, but we have never released this package. Is there something wrong in the changelog or I missed something?

Reply:
Changed